src/reducers/index.js

```diff
@@ -1,6 +1,11 @@
 import { combineReducers } from 'redux';
-import { charsReducer } from './starWarsReducer';
+
+import characters from './characters';
+import fetching from './fetching';
+import error from './error';
 
 export default combineReducers({
-  charsReducer
+  characters,
+  fetching,
+  error
```

Comment:
ah, very nice way of doing this to actually get to use the combineReducer to, well, combine multiple reducers. helps ensure you get the concept down and the syntax needed to reference your state trees within your MSTP